### PR TITLE
fix: progress bar issue

### PR DESCRIPTION
**Pull request by yikayiyo**
# Description

On the `/datasets` page, clicking on a delete icon triggers the unexpected appearance of the progress bar. The latest version of the `next-nprogress-bar` package provides a solution to this issue. However, it might be worth considering redesigning the `DatasetCard` component, as encapsulating numerous elements within a single `Link` component is not considered a best practice.

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Dependency upgrade

### CODE REVIEW

File: web/yarn.lock
Comment (crazywoola) [inline web/yarn.lock:1]: There is no actual changes of package.json, so this should not be committed into the vc system.

**Review by crazywoola — CHANGES_REQUESTED** [review]

**Comment by crazywoola** [comment]
Hello please add my wechat crazyphage I will invite you to the contributors' group.

File: web/yarn.lock
Comment (yikayiyo) [reply web/yarn.lock:1]: I ran `yarn upgrade next-nprogress-bar` and it didn't change the package.json file. Should I update the package.json file?

Comment (crazywoola) [reply web/yarn.lock:1]: Yes, in this case you should change the deps as well.

Comment (yikayiyo) [reply web/yarn.lock:1]: I've updated the deps.

**Review by crazywoola — APPROVED** [review]

**Pull request merged**